### PR TITLE
add iOS App Store link into preferences-window.ui

### PR DESCRIPTION
**Pull request by ijiki16**
image

close issues #1366

### CODE REVIEW

File: data/ui/preferences-window.ui [hunk 1]
```diff
@@ -574,6 +574,44 @@
                                 <property name="top_attach">1</property>
                               </packing>
                             </child>
+                            <child>
+                              <object class="GtkLabel" id="kdeconnect-ios">
+                                <property name="visible">True</property>
+                                <property name="can_focus">False</property>
+                                <property name="halign">end</property>
+                                <property name="label">iOS:</property>
```
Comment (ferdnyc) [inline data/ui/preferences-window.ui:582]: Hmm. I wonder if, just to be super clear (and avoid reports correcting us later), we should label this "iOS/iPadOS" or "iOS & iPadOS"? (I assume it works on iPads as well, no?)

I'm usually perfectly happy to call them both "iOS", but it seems like there's always someone turning up to point out, "iPads don't run iOS!" (Really I'm fine either way, though.)

**Review by ferdnyc — APPROVED** [review]

File: data/ui/preferences-window.ui (hunk 1 above)
Comment (ijiki16) [reply data/ui/preferences-window.ui:582]: I thought about the same thing, but everywhere in other parts of code & docs, it says `iOS`:  https://github.com/GSConnect/gnome-shell-extension-gsconnect/commit/9410f9a6b760dc0b10d515d3df76e5d541b7b036 , https://github.com/GSConnect/gnome-shell-extension-gsconnect/pull/1365/commits/efae3b6746a628e71a31c3c3f52ce9d5df424bc2. 
 I can easily change the label, but I think these parts must change too.

Comment (ferdnyc) [reply data/ui/preferences-window.ui:582]: @ijiki16 Mmm, fair point. Let's stick with this, then. If we decide to change it, we can change them all at once in a different PR.

**Comment by ferdnyc** [comment]
Are we too late for v54 on this one, @andyholmes ?

**Comment by andyholmes** [comment]
> Are we too late for v54 on this one, @andyholmes ?

Yeah, already released :/

**Pull request merged**